src/edu/cmu/tartan/Game.java

```diff
@@ -163,7 +167,23 @@ public void start() throws NullPointerException {
                     help();
                 } else if (input.compareTo("status") == 0) {
                     status();
-                } else {
+                } else if(input.split(" ")[0].compareTo("save") == 0){
+					printOutInterface.console("save....");
+					if(input.split(" ").length > 1){
+						int min = Integer.parseInt(input.split(" ")[1]);
+						if(min < 1 || min >10){
+							printOutInterface.console("save time period  allow 1 min to 10 min");
+							printOutInterface.console("save fail");
+							continue;
+						}
+						save(min*1000*60);
```

Comment:
This parameter is not readable when it is processed, and it is difficult for other developers to recognize it at the time of maintenance.
I think you need to handle "#define".